PhysicsTools/PatUtils/python/patPFMETCorrections_cff.py

```diff
@@ -138,6 +138,7 @@
     variation = cms.int32(0),  # If not specified, default to 0
 
     seed = cms.uint32(37428479),  # If not specified, default to 37428479
+    useDeterministicSeed = cms.bool(True),
```

Comment:
this does not match the PR description
> Currently I have the useDeterministicSeed option disabled by default

I suppose, the description should be updated.